### DISCUSSION TITLE
Flash R9mm without FC and STLink

### DISCUSSION THREAD

**Discussion by Justinas1980** [ExpressLRS/ExpressLRS#2555] [General]
Hello I successfuly flasher my R9M module to ELRS, and flashed R9mm receiver to ELRS Bootloader, but can't flash firmware to receiver, because I dont use BF FC (using Ardupilot) ant do not have STLint, why I cant flash via Radio like module?

**Comment by xagos** [comment]
you can flash the r9mm (after bootloader sport flashing) with serial passthrough (havent tested it with arducopter) or with any simple usb to ttl adapter...

**Comment by xagos** [comment]
This is a update setup without a FC...

![R9mm_ttl](https://github.com/ExpressLRS/ExpressLRS/assets/13057167/834ebd89-c12b-4255-975f-020cafc2a76c)
